### ISSUE TITLE
Undefined Columns Selected

### ISSUE THREAD

**Issue by tlmendez** [jgd29/CamCNV#1]
Greetings,

I am having trouble uploading the data for BAF, LRR, and Genotype. I keep getting "undefined columns selected".

I have exported the data from GenomeStudio in several ways. Is it possible to get an example of what the data should look like when uploading?

Some other bits of info. I am using data from the GSAv3 array demo data found on the illumina website. There are 654027 SNPs and 1 samples. I am using R 3.6.3 on the Ubuntu 20.04 platform. 

Thanks.

> ##### STEP 2 ####
> # 2. Load into datastore LRR and BAF for each sample at each probe
> #####
>
> nc <- nc_open("camcnv_onco.nc",write=TRUE)
> start_snp <- 1
> start_sample <- 1
> snp_length <- 654027
> sample_length <-12
> #Save our sample IDs in the NetCDF file
> sample_id_input <- read.csv("Samples Table.csv")
> ncvar_put(nc,"sampleID", sample_id_input$Sample_ID)
> #Create theB Allele Frequency and LRR input files by e.g. by exporting from Genome Studio Full Data Table
> #Ensure that the columns in the files ma .... [TRUNCATED]
> #Take out header columns
> baf_input <- as.matrix(baf_input[,4:27])
Error in `[.data.frame`(baf_input, , 4:27) : undefined columns selected
In addition: Warning messages:
1: replacing previous import ‘reader::cat.path’ by ‘NCmisc::cat.path’ when loading ‘bigpca’
2: replacing previous import ‘reader::get.ext’ by ‘NCmisc::get.ext’ when loading ‘bigpca’
3: replacing previous import ‘reader::rmv.ext’ by ‘NCmisc::rmv.ext’ when loading ‘bigpca’

**Comment by jgd29** [comment]
There are example data files uploaded with the code: baf_data_table.txt.gz, genotype_data_table.txt.gz,lrr_data_table.txt.gz
Basically three header columns and a column for each sample e.g.
Name    Chr     Position        9424085001_R01C01.Log R Ratio   9424085001_R10C01.Log R Ratio
rs3094315       1       752566  0.137246        0.1121343
rs3131972       1       752721  0.1064492       0.0665253

Are you only loading one sample? That could be the issue. All the samples should be loaded together. I see that the phrase "for each sample" in the instructions could be misleading here.

**Comment by tlmendez** [comment]
Greetings,

Thanks for the quick reply, so I have made the formatting of my BAF, LRR and genotype files to match, but I still get the same error. I have tried both csv and txt files.

I am adding all samples into each data file. So for example the BAF file has all 12 samples in it. I have attached a sample of the BAF file.

 I have replaced snpinfo with the locations of the GSAv3, and added all 12 samples to the samples id input. 

I wonder if I am missing something else. Any help will be appreciated. 

Thanks!
[BAF sample.txt](https://github.com/jgd29/CamCNV/files/6213964/BAF.sample.txt)
